### DISCUSSION TITLE
Is there any way to know if RSI is Bullish Divergences or Bearish Divergences ?

### DISCUSSION THREAD

**Discussion by SmallrookieQ** [DaveSkender/Stock.Indicators#906] [Q&A]
Is there any way to know whether the RSI has formed a bullish deviation or a bearish deviation?

**Comment by DaveSkender** [comment]
We've explored this concept of "divergence" as some form of an indicator (see #434) and found it to be a poor fit as an indicator for the library; but, still do respect the technical analysis concept of it.  Checkout #434 for some ideas, first.  Happy to explore ways to do this as well; which may be easier since we introduced chaining concepts in v2.

One of the prerequisites for this analysis was implemented in our [Pivots](https://dotnet.stockindicators.dev/indicators/Pivots/#content) indicator; it can be used to find these line segments on both price and RSI using chaining.

![image](https://user-images.githubusercontent.com/8432125/201400844-ae7f2dc8-e4c7-4228-add2-e0bd784043a6.png)

```csharp
var prcPivots = quotes
  .GetPivots();

var rsiPivots = quotes
  .GetRsi(..)
  .GetPivots();
```

**Reply by jimtollan** [reply]
maybe this use of pivots along with confirmation (you suggested adx in this [post](https://github.com/DaveSkender/Stock.Indicators/discussions/946)), could reap benefits...

will explore
